src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,103 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import React from 'react';
+import { Person, SortOptions } from '../types';
+import { PersonLink } from './PersonLink';
+import classNames from 'classnames';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+  const currentSort = searchParams.get('sort') || '';
+  const currentOrder = searchParams.get('order') === 'desc' ? 'asc' : 'desc';
```

Comment:
The `currentOrder` logic seems to toggle the order every time the component renders, which might not be the intended behavior. Consider using a function to determine the next order based on the current state, or ensure that the toggle only occurs when a sort option is clicked.